serviceinsight/upgrades/support-policy.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ServiceInsight support policy
+summary: Versions of ServiceInsight that are currently supported
+reviewed: 2022-08-05
+related:
+ - nservicebus/licensing
+ - nservicebus/upgrades/all-versions
+ - nservicebus/upgrades/support-policy
+ - nservicebus/upgrades/release-policy
+ - nservicebus/upgrades/supported-versions
+ - nservicebus/upgrades/supported-platforms
+ - servicecontrol/upgrades/support-policy
+ - servicecontrol/upgrades/supported-versions
+---
+
+Only the latest version of ServiceInsight is supported.
```

Comment:
Thinking more, I don't think we need a grid like we do with downstreams and ServiceControl. As long as it's "Only the latest version" then we can just point to what that version is.

```suggestion
Only the latest version of ServiceInsight is supported. The latest version can be found on the [downloads page](https://particular.net/downloads).
```

Then there's also no include file to have to keep up-to-date. (Until we find a reason that we really do need that.)